flake.nix

```
@@ -0,0 +1,41 @@
+{
+  description = "Delta Chat for Android";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
```

I suggest
```suggestion
    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
```
for stuff that's not related to NixOS.